extending/provider/implementation.rst

```diff
@@ -1,85 +1,44 @@
 Implementation
 --------------
 
-Annotations
-+++++++++++
+Every cloud provider is made up of one or more resources. Each resource implementation will manage the lifecycle
+of a resource in the cloud: create, read, update, and delete.
 
-**@Type(string)**
-    The class should be annotated with the ``@Type(string)`` annotation. The name provided by this annotation is
-    used by the Gyro language to lookup the resource implementation. For example, ``@Type("instance")`` in the AWS
-    provider will make ``aws::instance`` available to the Gyro language.
+A resource is any entity in a cloud provider that has state. For example, ``aws::load-balancer`` or
+``aws::instance`` are resources with state where-as a service like AWS Rekognition has no persistent state so
```

Comment:
I don't believe "where-as" is hyphenated. 
https://www.merriam-webster.com/dictionary/whereas